packages/react-component-library/src/components/TabSet/TabSet.tsx

```diff
@@ -35,6 +35,21 @@ export const TabSet: React.FC<TabSetProps> = ({
     }
   }
 
+  function handleKeyDown(event: KeyboardEvent<HTMLLIElement>) {
+    const { which } = event
+    let nextIndex
+
+    if (which === 37) {
```

Comment:
Rather than a `number` and then a comment on the next line, it could work well to add a constant and remove the comment.